service/service.go

```diff
@@ -12,8 +12,8 @@ import (
 	"net/http"
 	"strconv"
 
-	"github.com/cerbos/cerbos/client"
-	cerbos "github.com/cerbos/cerbos/client"
+	"github.com/cerbos/cerbos-sdk-go/cerbos"
+	client "github.com/cerbos/cerbos-sdk-go/cerbos"
```

Comment:
Why is the same package imported twice?

Reply:
If I don't import `client` as: `client "github.com/cerbos/cerbos-sdk-go/cerbos"`

It throws me the below error in `principalContext` function:
undefined: client
Line Number:
https://github.com/SuperAayush/demo-rest/blob/25244fac36fc4f50c2e8fe3c0f7013fc16ae91be/service/service.go#L159

Reply:
Taking reference from here for solving the above mentioned error: https://github.com/cerbos/cerbos-sdk-go/blob/aa488281f36f177ca3215c95b6b26949569c0add/README.md?plain=1#L41

Reply:
Instead of importing the package twice you need to change the package references in code.

Reply:
Made the changes please review.